servers/rendering/renderer_rd/shaders/scene_forward.glsl

```diff
@@ -787,13 +787,11 @@ LIGHT_SHADER_CODE
 
 #ifndef USE_NO_SHADOWS
 
-// Produces cheap white noise, optimized for window-space
-// Comes from: https://www.shadertoy.com/view/4djSRW
-// Copyright: Dave Hoskins, MIT License
+//Interleaved Gradient Noise
+//http://www.iryoku.com/next-generation-post-processing-in-call-of-duty-advanced-warfare
```

Comment:
```suggestion
// Interleaved Gradient Noise
// http://www.iryoku.com/next-generation-post-processing-in-call-of-duty-advanced-warfare
```

We should keep `//code` for code that is commented out - for actual documentation, the added space makes it clear that it's text and not code.

Reply:
Also, the URL is "Server Not Found".

Reply:
Thanks, fixed!